### ISSUE TITLE
"Load More" button should refocus on new items to help screen readers

### ISSUE THREAD

**Issue by echappen** [ustaxcourt/ef-cms#598]
When using a screen reader on Results table, the "Load More" button does not maintain focus on listed content, nor does it signal that loading has completed. 

#### Why This Matters
People who rely on keyboard navigation or screen readers can become easily confused when focus is lost, or no success/failure message is given for asynchronous actions. See Focus Management section [of this article](https://www.smashingmagazine.com/2015/05/client-rendered-accessibility/) for more info. 

#### Possible solutions
1. When loading is complete, refocus back to first item of newly loaded content
2. Provide screen reader-only alert message when loading is complete
2. Change lazy loading to pagination

**Comment by JessicaMarine** [comment]
Moved to Flexion: https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/flexion/ef-cms/8102
